source/overloading.tex

```diff
@@ -2484,18 +2475,18 @@
 \pnum
 This subclause defines a partial ordering of implicit conversion
 sequences based on the relationships
-\term{better conversion sequence}
+\defnx{better conversion sequence}{conversion!sequence!better}
```

Comment:
I think we should keep "conversion sequence" separate from "conversion" in the index, since they have quite different sets of rules.

Reply:
Fixed.